ConfidenceDemoApp/src/main/java/com/example/confidencedemoapp/MainVm.kt

```diff
@@ -100,16 +85,14 @@ class MainVm(app: Application) : AndroidViewModel(app) {
 
     fun updateContext() {
         val start = System.currentTimeMillis()
-        ctx = ImmutableContext(
-            attributes = mutableMapOf(
-                "user_id" to Value.String(UUID.randomUUID().toString()),
-                "picture" to Value.String("hej"),
-                "region" to Value.String("eu")
-            )
+        val ctx = mapOf(
+            "user_id" to ConfidenceValue.String(UUID.randomUUID().toString()),
+            "picture" to ConfidenceValue.String("hej"),
+            "region" to ConfidenceValue.String("eu")
         )
         viewModelScope.launch {
             Log.d(TAG, "set new EvaluationContext")
-            OpenFeatureAPI.setEvaluationContext(ctx)
+            confidence.putContext(ctx)
```

Comment:
Do we have any way to know when the new values are available for the app to consume? 
In my testing the `refreshUI` that follows this line is always returning the older STALE value